### PR TITLE
added extra page for address safety review

### PR DESCRIPTION
**Pull request by whitfield-mj**

### CODE REVIEW

File: server/views/curfew/addressSafety.pug [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    div.pure-g.pure-u-1
+        div.borderBottomLight
+            div.back-link-container.smallPaddingTop
+                a.link-back(href="javascript: window.history.back();") Back
+
+            h2.heading-large Proposed curfew address
+
+            if address
+                p The following discharge address have been requested by the offender.
+
```
Comment (atodd-noms) [inline server/views/curfew/addressSafety.pug]: The following address has been requested by the offender.

File: server/views/curfew/addressSafety.pug [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    -var addressSafety = data.curfew && data.curfew.addressSafety ? data.curfew.addressSafety : {}
+    -var address = data.proposedAddress && data.proposedAddress.curfewAddress ? data.proposedAddress.curfewAddress.preferred : {}
+
+    div.pure-g.pure-u-1
+        div.borderBottomLight
+            div.back-link-container.smallPaddingTop
+                a.link-back(href="javascript: window.history.back();") Back
+
+            h2.heading-large Proposed curfew address
+
```
Comment (atodd-noms) [inline server/views/curfew/addressSafety.pug]: Proposed address decision

File: server/views/curfew/addressSafety.pug [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        -var pendingText = "Yes - pending confirmation of risk management planning"
+        form(method="post")
+            input(type="hidden" name="_csrf" value=csrfToken)
+            input(type="hidden" name="nomisId" value=nomisId || '')
+            div.pure-g.largePaddingBottom
+                div.pure-u-1.pure-u-md-1-2
+                    div.form-group
+                        p Could the offender be managed safely at this address?
+
```
Comment (atodd-noms) [inline server/views/curfew/addressSafety.pug]: this offender

**Comment by whitfield-mj** [comment]
Text errors fixed

**Pull request merged**